src/App.js

```diff
@@ -1,22 +1,18 @@
 import logo from './logo.svg';
 import './App.css';
+import Greeting from './components/pure/greeting';
+import GreetingF from './components/pure/greetingF';
+import TaskListComponent from './components/container/task_list';
 
 function App() {
```

Comment:
It would be a good practice to call the main component as `Home`. Then a good practice could be to have the folders like `src/Home/index.js`.
We usually dont call the js files with the name of the component. Insted we have a folder with that name and a `index.js` file